### ISSUE TITLE
Error: Cannot find an available port

### ISSUE THREAD

**Issue by SNAnalyst** [jhelvy/renderthis#79]
I regularly need to convert Xaringan presentations to pdf, using r`enderthis::to_pdf("name.html", complex_slides = TRUE, partial_slides = TRUE, delay = 5) `. This used to work flawlessly.

Since about two weeks I keep getting the error "Cannot find an available port". First, it only happened on one laptop, but since a week it is now happening on both laptops. So, this does not seem specific to my own setup. Also, a colleague of mine gets the same error.
Interesting, there seem to be at least two potential causes, that are shown:

```
Error in `with_random_port()`:
! Cannot find an available port. Please try again.
Caused by error in `Sys.getenv()`:
! wrong type for argument
---
Backtrace:
     ▆
  1. └─renderthis::to_pdf(...)
  2.   └─renderthis:::to_pdf_complex(...)
  3.     └─chromote::ChromoteSession$new()
  4.       └─chromote (local) initialize(...)
  5.         └─chromote::default_chromote_object()
  6.           ├─chromote::set_default_chromote_object(Chromote$new())
  7.           └─Chromote$new()
  8.             └─chromote (local) initialize(...)
  9.               └─Chrome$new()
 10.                 └─chromote (local) initialize(...)
 11.                   └─chromote:::launch_chrome(path, args)
 12.                     └─chromote:::with_random_port(launch_chrome_impl, path = path, args = args)
 13.                       └─rlang::abort(...)
```

OR

```
Error in `with_random_port()`:
! Cannot find an available port. Please try again.
Caused by error in `startup()`:
! Failed to start chrome. Error: 
---
Backtrace:
     ▆
  1. └─renderthis::to_pdf(...)
  2.   └─renderthis:::to_pdf_complex(...)
  3.     └─chromote::ChromoteSession$new()
  4.       └─chromote (local) initialize(...)
  5.         └─chromote::default_chromote_object()
  6.           ├─chromote::set_default_chromote_object(Chromote$new())
  7.           └─Chromote$new()
  8.             └─chromote (local) initialize(...)
  9.               └─Chrome$new()
 10.                 └─chromote (local) initialize(...)
 11.                   └─chromote:::launch_chrome(path, args)
 12.                     └─chromote:::with_random_port(launch_chrome_impl, path = path, args = args)
 13.                       └─rlang::abort(...)
```

Again, not one specific cause, but it always happens in the interaction between `renderthis::to_pdf` and `chromote:::with_random_port`. 

Considering that it all worked well on my second laptop and I am now getting the same error there, it seems plausible that it has to do with either an update in r`enderthis` or `chromote` or in Chrome itself.

**Comment by gadenbuie** [comment]
Make sure you're using the latest version of chromote. Both issues are solved in v0.3.1.

**Comment by SNAnalyst** [comment]
Thanks, that fixed it indeed. I was using 0.3.0
